Formula/pyqt-builder.rb

```diff
@@ -15,6 +15,26 @@ class PyqtBuilder < Formula
   depends_on "python@3.9"
   depends_on "sip"
```

Comment:
Aren't the resources here already covered by `depends_on "sip"`?

Reply:
They should be, yes.

Reply:
👍